api/indexer.oas2.json

```diff
@@ -760,6 +770,16 @@
           "description": "The application which this local state is for.",
           "type": "integer"
         },
+        "opted-in-at-round": {
+          "description": "Round when the account opted into the application.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "closed-out-at-round": {
+          "description": "Round when account closed out of the application.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
```

Comment:
ApplicationLocalState:
* OptedInAtRound - because you set OnComplete to the OPTIN enum to add an application to an account.
* ClosedOutAtRound - because you set OnComplete to the CLOSEOUT enum to remove an application from an account.

goal app -h
```
Available Commands:
  call        Call an application
  clear       Clear out an application's state in your account
  closeout    Close out of an application
  create      Create an application
  delete      Delete an application
  info        Look up current parameters for an application
  interact    Interact with an application
  optin       Opt in to an application
  read        Read local or global state for an application
  update      Update an application's programs
```